pkg/service/service_suite_test.go

```diff
@@ -1,13 +1,294 @@
-package service_test
+package service
 
 import (
+	"context"
+	"fmt"
+	"os"
 	"testing"
 
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"k8s.io/utils/mount"
+
+	"github.com/kubevirt/csi-driver/pkg/kubevirt"
 )
 
+const serialID = "4b13cebc-7406-4c19-8832-7fcb1d4ac8c5"
+
 func TestService(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Service Suite")
 }
+
+var _ = Describe("NodeService", func() {
```

Comment:
I ,personally , prefer the Go testing package. I find it simpler and more readable.